### PR TITLE
Np 2511 add testing

### PR DESCRIPTION
**Pull request by svenngjeto**

### CODE REVIEW

File: src/tests/unit/handler.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  const event = {
+    query: {
+      query: 'journal',
+      year: 2021,
+      start: 1
+    }
+  }
+
+  describe('Test search response content', () => {
+    it('verifies response is success and has a empty body', async function () {
```
Comment (axthosarouris) [inline src/tests/unit/handler.test.js:17]: The body is not checked that is empty and it is not empty.

**Review by axthosarouris — CHANGES_REQUESTED** [review]
github actions file is missing.

File: src/tests/unit/handler.test.js (hunk 1 above)
Comment (svenngjeto) [reply src/tests/unit/handler.test.js:17]: Now it checks that the body is a empty object

Comment (axthosarouris) [reply src/tests/unit/handler.test.js:17]: expect(response.body).to.be.an('Object').that.is.empty or expect(response.body).to.be.empty

**Review by axthosarouris — APPROVED** [review]
Approved provided that the next PR will be about running tests in github

**Pull request merged**